industrial_ci/src/tests/source_tests.sh

```diff
@@ -23,6 +23,8 @@
 # (As of version 0.4.4 most of them are defined in env.sh).
 
 function install_catkin_lint {
+    ici_install_pkgs_for_command apt-add-repository software-properties-common
+    ici_asroot apt-add-repository ppa:roehling/ros -y -u # -u for versions pre-Ubuntu 18.04
```

Comment:
using a PPA would fail for Debian

Reply:
I could just install it from pip. would that be okay? I know generally this is frowned upon for ros, but since this isn't a ros package and is in an isolated ci environment, I think it'd be a valid solution to fix the debian problem.

Reply:
You mean, you want to revert https://github.com/ros-industrial/industrial_ci/pull/448? ;)
But the main problem persists: The user might use the catkin_lint version from the ROS repository..
Right now I do not see a string demand for the the latest catkin_lint version.

And you can always install it with pip, e.g. in `AFTER_INIT`.
`ici_install_pkgs_for_command` will not install anything, if the command is already available.
It just exists for the purpose of installing a different version beforehand.

Reply:
> Right now I do not see a strong demand for the the latest catkin_lint version.

I could add a :+1: here (as in: I'd like to use the latest version).

It's the reason I opened ros/rosdistro#24251.

There's little value in having a linter available if you don't have access to the latest version.

Reply:
I am open to reverting #448 (for master only), but someone needs to fix the catkin_lint issues and document it properly ;)
But we might want to run a test beforehand to see how many regressions we will get.

As an alternative, `AFTER_INIT='pip install catkin-lint'` (not tested..) should work for everyone right now.

Reply:
`AFTER_INIT='pip install catkin-lint'` would be an opt-in, `ADDITIONAL_DEBS=python-catkin-lint` would be an opt-out (if we make pip the default).

Reply:
>  but someone needs to fix the catkin_lint issues and document it properly ;)

Which issues do you refer to here?

> As an alternative, `AFTER_INIT='pip install catkin-lint'` (not tested..) should work for everyone right now.

I agree, but seeing as `CATKIN_LINT` is something users can "just enable", it seems like not too strange for them to expect that would lint their packages using the latest released version.

Right now it doesn't do that, but only because OR has stopped importing the `.deb` from @roehling's PPA. That's not very clear.

Reply:
I can change this pr to instead revert #448, fix catkin_lint issues in tests triggered by the new version, and update docs with the addition deb opt-out. 

I'm very busy rn, so I'll push this till the weekend 😅

Reply:
> This opt-out needs a ici_install_pypi_for_command (or some better name) to not install catkin_lint, if it is already installed.

I doesn't have to be a separate command for now, but it would be appreciated..

Reply:
@ipa-mdl Is what I most recently pushed match what you're looking for?